config/bridge.go

```diff
@@ -17,17 +17,20 @@
 package config
 
 import (
+	"github.com/limechain/hedera-eth-bridge-validator/app/domain/service"
 	"github.com/limechain/hedera-eth-bridge-validator/config/parser"
 	"github.com/limechain/hedera-eth-bridge-validator/constants"
 	log "github.com/sirupsen/logrus"
 	"math/big"
 )
 
 type Bridge struct {
-	TopicId string
-	Hedera  *BridgeHedera
-	EVMs    map[uint64]BridgeEvm
-	Assets  Assets
+	TopicId          string
+	Hedera           *BridgeHedera
+	Assets           service.Assets
```

Comment:
Since Assets is no longer a config, but a service, wouldn't be best to remove it from here?